data_models/website.py

```diff
@@ -7,13 +7,17 @@
 class Website:
     """This class stores the information about a website"""
 
-    __LOGO_API = 'https://www.google.com/s2/favicons?domain='
+    __LOGO_API = 'https://logo.clearbit.com/'
```

Comment:
This will change all logos, right?

Reply:
Do we want to change logo for rest of the stamp pages? If yes, then do we need to mention it?

Reply:
Mention it where? We are giving attribution for all logos to clearbit at end. I guess that will do it.

Reply:
Basically, I changed the API here also because if we don't, then we will be using two different APIs for same purpose (getting logo).

Reply:
Got it. That should be fine.